olive/engine/engine.py

```diff
@@ -919,7 +919,7 @@ def _run_pass(
         output_model_path = str(output_model_path)
 
         # prune if invalid search_point
-        if not p.validate_search_point(pass_search_point) and not self.no_search:
+        if not p.validate_search_point(pass_search_point, accelerator_spec, with_fixed_value=True):
```

Comment:
`with_fixed_value` should the default value be False here ?

Reply:
If `False`(legacy way actually), we cannot validate the search point with fixed value. Let us say:
fp16 is a fixed value and user set is as True. Then if `with_fixed_value` is True, we can use the value of fp16 to filter CPU ep out.